digitalstrategie/assets/scss/_base.scss

```diff
@@ -1,22 +1,78 @@
+*,
+*:before,
+*:after {
+    box-sizing: inherit;
+}
+
 //fixes horizontal scroll issue cause by header+footer width
-html,
-body {
+html {
+    overflow-y: scroll;
+    background-color: $body-bg;
+    color: contrast-color($body-bg);
+    box-sizing: border-box;
     font-family: $font-family-base;
+    font-family: $font-family-sans-serif;
+    line-height: 1.4;
+
     scroll-behavior: smooth;
-    // scroll-padding-top: 4rem; might need
+
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+
+    // overflow-x: hidden;  // stops horizontal scroll
+}
+
+body {
+    margin: 0;
+}
+
+p,
+address,
+blockquote,
+pre,
+table,
+dl,
+ul,
+ol,
+figure,
+fieldset {
+    margin-top: 0;
+    margin-bottom: $spacer;
 }
 
-//reset to be more flexible for headings
 h1 {
-    margin: 0
+    font-size: $h1-font-size;
 }
 
 h2 {
-    font-size: 1.1rem;
-    font-weight: inherit;
```

Comment:
same as above, we can't have all h2 at a small size, it's better to add a utility or a specific class for these use cases, base should be used for ensuring defaults not for styling elements